providers/amazon/docs/connections/exasol.rst

```diff
@@ -0,0 +1,174 @@
+.. _exasol_airflow_setup:
```

Comment:
Please do not duplicate documentation. Plus, this documentation is related to Exasol, it has nothing to do with Amazon. As I mentioned in my previous comment, this should go in Exasol documentation